realtime_ospl_c/build.sh

```diff
@@ -0,0 +1,13 @@
+rm -fr c
+mkdir c
+cd c
+OSPL_TMPL_PATH=/usr/etc/opensplice/idlpp idlpp -S -l c ../LargeMsg.idl
+echo "Generated C code under '`pwd`'"
+cd ..
+
+rm -fr build
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=~/local
```

Comment:
This build script was specialized for my build environment, so I decided to make it more flexible. In b09eb99e7f84b112efdb6863acce3282af4bb0a2 I modified this script to take pass arguments directly to the cmake command, so you can configure the build with `. build.sh -DCMAKE_BUILD_TYPE=Release`.